sim/warrior/arms/arms.go

```diff
@@ -39,28 +42,95 @@ func NewArmsWarrior(character *core.Character, options *proto.Player) *ArmsWarri
 		Options: armsOptions,
 	}
 
-	// rbo := core.RageBarOptions{
-	// 	StartingRage:   armsOptions.ClassOptions.StartingRage,
-	// 	RageMultiplier: core.TernaryFloat64(war.Talents.EndlessRage, 1.25, 1),
-	// }
-	// if mh := war.GetMHWeapon(); mh != nil {
-	// 	rbo.MHSwingSpeed = mh.SwingSpeed
-	// }
-	// if oh := war.GetOHWeapon(); oh != nil {
-	// 	rbo.OHSwingSpeed = oh.SwingSpeed
-	// }
+	rbo := core.RageBarOptions{
+		StartingRage: armsOptions.ClassOptions.StartingRage,
+		OnHitDealtRageGain: func(sim *core.Simulation, spell *core.Spell, result *core.SpellResult, rage float64) float64 {
+			return rage * 1.25 // Endless Rage is now part of Anger Management, now an Arms specialization passive
+		},
+	}
+	if mh := war.GetMHWeapon(); mh != nil {
+		rbo.MHSwingSpeed = mh.SwingSpeed
+	}
+	war.EnableRageBar(rbo)
```

Comment:
This multiplier can just be passed down as a multiplier to the rage bar instead of having to do a complete new callback every time you do a hit. It seems like an overkill for just a static 25% increased rage gen on hits.